src/AVFoundation/AVVideoSettings.cs

```diff
@@ -98,11 +100,12 @@ public AVVideoSettingsUncompressed (NSDictionary dictionary)
 
 		public AVVideoScalingMode? ScalingMode {
 			get {
-				return ScalingModeFromNSString (GetNSStringValue (AVVideo.ScalingModeKey));
+				var val = GetNSStringValue (AVVideo.ScalingModeKey);
+				return val is not null ? ScalingModeFromNSString (val) : null;
```

Comment:
I think it's simpler to change `ScalingModeFromNSString` to take a nullable string (in which case this change would not be needed at all).